doc-content/kogito-docs/src/main/asciidoc/creating-running/con_kogito-automation.adoc

```diff
@@ -31,11 +31,11 @@ For example, {PRODUCT} is compatible with the following technologies:
 * *GraalVM* enables you to use native compilation with {PRODUCT}, resulting in fast start-up times and minimal footprint. For example, a native {PRODUCT} service starts in about 0.003ms, about 100 times faster than a non-native start-up. Fast start-up is almost a necessity in a cloud ecosystem, especially if you need small serverless applications.
```

Comment:
We should take care about it, on Quarkus, AFAIK native-builds will not be supported for now.

Reply:
Thanks @spolti. @evacchi, @ricardozanini, can someone confirm this? This is kind of a big deal and I hadn't heard this from anyone else yet. Just confirming.

Reply:
I meant, will not be supported on Quarkus product, for community we're free to use it.

Reply:
@spolti, so I'm not sure what you want me to say here then.

@evacchi, wdyt?

Reply:
confirm. we could just conditionally hide this in dev preview doc

Reply:
@spolti, sorry, I get it now. Makes sense. Will condition this for community only. Thanks.